src/components/ui/ImageGallery/ImageGallerySelector.tsx

```diff
@@ -57,7 +62,7 @@ function ImageGallerySelector({ images, onSelect, currentImageIdx }: Props) {
         <IconButton
           aria-label="backward slide image selector"
           icon={<Icon name="ArrowLeft" width={24} height={24} />}
-          onClick={() => moveScroll(elementsRef.current, -200)}
+          onClick={() => moveScroll(elementsRef.current, -400)}
```

Comment:
even if it is a temporary workaround, I think would be good to create a static const for this value to avoid magic numbers in the code. Maybe a `const SCROLL_TOP_VALUE =  400` on the top of the component, what do you think?

```suggestion
onClick={() => moveScroll(elementsRef.current, -(SCROLL_TOP_VALUE))}
```

Reply:
Nice! I'll add it!